alumnium/agents/verifier_agent.py

```diff
@@ -78,11 +74,4 @@ def invoke(self, statement: str, vision: bool = False) -> Verification:
         logger.info(f"  <- Reason: {verification.explanation}")
         logger.info(f'  <- Usage: {message["raw"].usage_metadata}')
 
-        if not verification.result:
-            loading = self.loading_detector_agent.invoke(aria, title, url, screenshot)
-            if loading and self.retry_count > 0:
-                sleep(LoadingDetectorAgent.delay)
-                self.retry_count -= 1
-                return self.invoke(statement, vision)
```

Comment:
@p0deje I'm not sure about this change. I'm thinking of keeping it like this. wdyt?
```python
        if not verification.result and self.retry_count > 0:
                sleep(LoadingDetectorAgent.delay)
                self.retry_count -= 1
                return self.invoke(statement, vision)
```